### PR TITLE
Redo basic page website 

### PR DESCRIPTION
**Pull request by Magons**
Created mockup based on Paul's comments. https://jetthoughts.mybalsamiq.com/projects/oauth2me/grid

The setup process should be represented with the help of pictures. For reference: 

f8c3d362-b846-11e4-811e-c0f1cd4be817
20f77d98-b847-11e4-81d7-e9ab3bf92224

### CODE REVIEW

**Comment by Magons** [comment]
@nazarky and @tehhi42: pls [verify](http://development.ipr-oauth2-me.divshot.io)

**Comment by pftg** [comment]
@Magons cannot verify becasue of 502 error: http://note.io/1MK2LuM

**Comment by Magons** [comment]
@pftg  hm..I open it without problems. Try [this](http://development.oauth2me.divshot.io/)

**Comment by pftg** [comment]
the same

**Comment by NazarKY** [comment]
@pftg: Everything works, no 502 error.

File: divshot-debug.log [hunk 1]
```diff
@@ -0,0 +1 @@
...b3b29ce8e3","assets/javascripts/URI.js":"aa6de429373d5bea62c36ed100e98e529aa9dbb593eccfd73aac43a4e0eaf803","assets/javascripts/base64.js":"bf4e16e161930eced85a401f4bafad466795de48bd58350c5129e201da04c3a6","assets/javascripts/bootstrap.js":"20f9dc1b8566a371cc3e871516427ca49165c3c5ee486801240fe3e725da235c","assets/javascripts/chart.js":"6c630fb54a66f1094fe834d9e89ce928f8d847d530bef93cc8985714ea148877","assets/javascripts/redirect.coffee":"eb490dc5bce521e7f3dacd1bd5b0b583e4624f72ccf8773820fbd1ed511888c7","assets/stylesheets/bootstrap.css":"2bc23b04ee22bb8fb16a47076dbb82b2a06965cafa40d53f5ab1f4bb84e3cf22","assets/stylesheets/font-awesome.min.css":"e257f06dc6e7f1627fc283c6f1ec1c326872e7037fa39f0f7b67e429cfe43c34","assets/stylesheets/main.css":"3ccdb0c5e2d84ca550e0f97ab49b64da4280b919082f46865d1c6396544dc404","assets/stylesheets/main.sass":"54fa5c869c481522fac3e717b721e16faeb55a813207e107d5d7ba0f05190905","divshot.json":"ada71f88cfdb91220ec8d20a3c51a1e60bab36f463bece9712099ce19faf80c8","index.html":"bd37d8a22df86830846026fe4b35193cae2b4eacdd7e6bdeec6044aa647d4204","redirect/index.html":"f1da9dbd69c050d80b5c9687f5b073d969e9e89dab41734bf886ad2620615509"},"file_mapped":true,"loadpoint":{}}}
```
Comment (NazarKY) [inline divshot-debug.log]: Don't push this file. Add it to gitignore.

**Pull request merged**